src/compiler/visitor.ts

```diff
@@ -268,7 +268,7 @@ namespace ts {
                     nodesVisitor((<PropertyDeclaration>node).decorators, visitor, isDecorator),
                     nodesVisitor((<PropertyDeclaration>node).modifiers, visitor, isModifier),
                     visitNode((<PropertyDeclaration>node).name, visitor, isPropertyName),
-                    visitNode((<PropertyDeclaration>node).questionToken, tokenVisitor, isToken),
+                    (visitNode((<PropertyDeclaration>node).questionToken, tokenVisitor, isToken), visitNode((<PropertyDeclaration>node).exclamationToken, tokenVisitor, isToken)),

```

Comment:
This is a different style from the rest of the method - can you change / justify / add a comment?

Reply:
Because you can only have one or the other, the following might be more concise:
```suggestion
                    visitNode((<PropertyDeclaration>node).questionToken || (<PropertyDeclaration>node).exclamationToken, tokenVisitor, isToken),
```